components/compliance-service/inspec-agent/runner/runner.go

```diff
@@ -476,6 +499,23 @@ func (t *InspecJobSummaryTask) Run(ctx context.Context, task cereal.Task) (inter
 		return nil, errors.Wrap(err, "could not unmarshal summary job parameters")
 	}
 
+	for _, jobID := range jobsPayload.LostJobIDs {
+		endTime := time.Now()
+		startTime := jobsPayload.StartTime
+		if startTime.IsZero() {
+			startTime = endTime
+		}
+		job := &types.InspecJob{
+			InspecBaseJob: types.InspecBaseJob{
+				JobID: jobID,
```

Comment:
Can the node id be included as well?

not including it results in all rows with this error opening at once; see screenshot:
<img width="1266" alt="Screen Shot 2019-08-12 at 21 19 40" src="https://user-images.githubusercontent.com/10341541/62912879-14c08600-bd47-11e9-90eb-16ec1eb6f007.png">